tests/integration/deploy/test_deploy_command.py

```diff
@@ -52,7 +53,7 @@ def test_package_and_deploy_no_s3_bucket_all_args(self, template_file):
 
             self.assertEqual(package_process.process.returncode, 0)
 
-            stack_name = "a" + str(uuid.uuid4()).replace("-", "")[:10]
+            stack_name = f"test-package-and-deploy-no-s3-bucket-all-args-{CFN_PYTHON_VERSION_SUFFIX}"
```

Comment:
can we do something like

https://docs.python.org/3/library/unittest.html#unittest.TestCase.id

instead?

Reply:
Man, I was looking for this but was searching for `how to get current method name` :) I'll make the change.